README.md

```diff
@@ -271,33 +275,14 @@ workflow file (the `on.push.branches` config key). This is the recommended
 approach as it ensures only a single build per push.
 
 Alternatively, if it is important to you that workflows are triggered by pushes
-from the mirror workflow, you can accomplish this by supplying an alternative
-SSH key to the [checkout action][checkout-action]:
-
-1. [Generate a new SSH key][generate-ssh-key] locally. Don't worry about adding
-   it to the ssh-agent.
-
-2. Find the generated *public key* and [add it as a deploy key][add-deploy-key]
-   to the repository. Be sure to select "Allow write access".
-
-3. Find the generated *private key* and [add it as a secret][add-secret] to the
-   repository.
-
-4. Change the "Checkout" step in the mirror workflow to use the new deploy key:
-
-   ```yaml
-   - name: Checkout
-     uses: actions/checkout@v2
-     with:
-       ssh-key: ${{ secrets.DEPLOY_KEY }}
-   ```
-
-   Here, `DEPLOY_KEY` is the name you picked from step 3.
-
-With this, the mirror workflow will authenticate with GitHub using the deploy
-key instead of the default token when pushing commits, triggering any workflows
-as if a regular user had pushed those commits. This does not change the author
-or committer on the commits.
+from the mirror workflow, you can accomplish this by supplying a personal
+access token to the [Mirror Branch action][mirror-branch-action] by following
+the steps in the [previous section](#interaction-with-branch-protection).
+
+With this, the mirror workflow will authenticate with GitHub using the personal
+access token when updating commits, triggering any workflows as if a regular
+user had pushed those commits. This does not change the author or committer on
+the commits.
```

Comment:
This one is definitely citation needed... I don't actually know for a fact that going through the GitHub API will trigger the push event. @zofrex, do you happen to know? If not, I can try to test it this weekend.

Reply:
As far as I know there won't be any difference, but I'm not exactly familiar with the various Github events. I did some practical testing today and Travis builds and Netlify deploys are triggering as expected by the mirror action's update to the master branch. The mirror action also seems to work fine when the push to the base branch is made by a workflow, as it still fires when Bors is the one to merge a PR.

Reply:
I just noticed this in [the Github docs](https://docs.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token):

> When you use the repository's GITHUB_TOKEN to perform tasks on behalf of the GitHub Actions app, events triggered by the GITHUB_TOKEN will not create a new workflow run. This prevents you from accidentally creating recursive workflow runs. For example, if a workflow run pushes code using the repository's GITHUB_TOKEN, a new workflow will not run even when the repository contains a workflow configured to run when push events occur.

I think this means that although the mirror action triggered CI/CD builds via Travis and Netlify in my testing, it wouldn't trigger any CI/CD actions set up via GitHub Actions?